HyperVExtension/src/HyperVExtension/Templates/ReviewFormForVMGallery.json

```diff
@@ -0,0 +1,100 @@
+{
+  "type": "AdaptiveCard",
+  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+  "version": "1.6",
```

Comment:
Pretty sure this will refuse to render, you have to make it version 1.5

Reply:
Interestingly it renders perfectly with 1.6 still.